example/usps_api/server.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+#include "config/config_parser.h"
+
+#include <string>
+#include <iostream>
+#include <grpc/grpc.h>
+#include <grpcpp/server.h>
+#include <grpcpp/server_builder.h>
+#include <grpcpp/server_context.h>
+#include <grpcpp/security/server_credentials.h>
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+#include "proto/usps_api/sfc.grpc.pb.h"
+
+ABSL_FLAG(std::string, HOST, "localhost", "The host of the ip to listen on");
+// port will be within range due to flag definition
```

Comment:
Nit: In general, comments should be proper sentences.

Reply:
Fixed!